pkg/virt-operator/resource/apply/reconcile.go

```diff
@@ -1236,20 +1245,28 @@ func (r *Reconciler) deleteObjectsNotInInstallStrategy() error {
 	return nil
 }
 
-func (r *Reconciler) exportProxyEnabled() bool {
+func (r *Reconciler) isFeatureGateEnabled(featureGate string) bool {
 	if r.kv.Spec.Configuration.DeveloperConfiguration == nil {
 		return false
 	}
 
 	for _, fg := range r.kv.Spec.Configuration.DeveloperConfiguration.FeatureGates {
-		if fg == virtconfig.VMExportGate {
+		if fg == featureGate {
 			return true
 		}
 	}
 
 	return false
 }
 
+func (r *Reconciler) exportProxyEnabled() bool {
+	return r.isFeatureGateEnabled(virtconfig.VMExportGate)
+}
+
+func (r *Reconciler) commonInstancetypesDeploymentEnabled() bool {
+	return r.isFeatureGateEnabled(virtconfig.CommonInstancetypesDeploymentGate)
+}
+
```

Comment:
nit: should we include the ClusterConfig and use `CommonInstancetypesDeploymentEnabled()` function?